gnd/src/main/java/com/google/android/gnd/system/CameraManager.java

```diff
@@ -73,30 +75,26 @@ private Completable sendCapturePhotoIntent() {
   public Observable<Bitmap> capturePhotoResult() {
     return activityStreams
         .getNextActivityResult(CAPTURE_PHOTO_REQUEST_CODE)
-        .flatMap(this::onCapturePhotoResult);
+        .flatMapMaybe(this::onCapturePhotoResult);
```

Comment:
Should this method maybe return a `Maybe` instead?

Reply:
Done